assembl/static2/js/app/components/common/profileLine.jsx

```diff
@@ -1,39 +1,52 @@
+// @flow
 import React from 'react';
 import { Translate } from 'react-redux-i18n';
 import moment from 'moment';
+
+// $FlowFixMe
```

Comment:
out of curiosity, why $FlowFixMe ?

Reply:
Yes...We only need to configure flow here